code/modules/surgery/organs/blood.dm

```diff
@@ -40,52 +41,47 @@
 
 
 		//Effects of bloodloss
-		var/oxy_immune = species.flags & NO_BREATHE //Some species have blood, but don't breathe; they should still suffer the effects of bloodloss.
-
+		var/word = pick("dizzy","woozy","faint")
 		switch(blood_volume)
 			if(BLOOD_VOLUME_OKAY to BLOOD_VOLUME_SAFE)
 				if(prob(5))
-					to_chat(src, "<span class='warning'>You feel [pick("dizzy","woozy","faint")].</span>")
-				if(oxy_immune)
-					adjustToxLoss(round((BLOOD_VOLUME_NORMAL - blood_volume) * 0.01, 1))
-				else
-					adjustOxyLoss(round((BLOOD_VOLUME_NORMAL - blood_volume) * 0.01, 1))
+					to_chat(src, "<span class='warning'>You feel [word].</span>")
+				apply_damage_type(round((BLOOD_VOLUME_NORMAL - blood_volume) * 0.01, 1), species.blood_damage_type)
```

Comment:
the damage amount hasn't been changed; it's just using a different proc and the damage type is based upon a species var now, which will default to oxygen damage in all cases (but slimes).

Still, this could allow for neat things in the future like...say, giving IPCs exotic blood of oil.